### PR TITLE
[DOC]: API Reference for DevSettings.addMenuItem() #1523

### PR DESCRIPTION
**Pull request by mohamedsgap**
Started writing the doc for addMenuItem() as DevSettingsModule.

### CODE REVIEW

**Comment by mohamedsgap** [comment]
Hey @rickhanlonii just need you to review DevSettings API reference #1539 to see if i  wrote  it correctly or need some changes!

**Comment by mohamedsgap** [comment]
any updates ? @rickhanlonii

**Comment by mohamedsgap** [comment]
This pull request solved issue ref #1523

File: docs/devsettings.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+id: devsettings
+title: DevSettings
+---
+
+`DevSettings`
+
+---
+
```
Comment (rickhanlonii) [inline docs/devsettings.md]: Could we add a summary here? Something like "The DevSettings module exposes methods for customizing settings in for developers in development."

File: docs/devsettings.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# Reference
+
+## Methods
+
+### `addMenuItem()`
+
+```jsx
+ addMenuItem(title: string, handler: () => )
+```
+It allows users to add additional dev menu options to their app, so you can use following code to show secret dev screen
```
Comment (rickhanlonii) [inline docs/devsettings.md]:
```suggestion
Add a custom menu item to the developer menu:
```

**Comment by mohamedsgap** [comment]
Just made an update according to you suggestion, Let's merge the PR :) @rickhanlonii

File: docs/devsettings.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+id: devsettings
+title: DevSettings
+---
+
+The `DevSettings` module exposes methods for customizing settings in for developers in development.
```
Comment (rickhanlonii) [inline docs/devsettings.md]:
```suggestion
The `DevSettings` module exposes methods for customizing settings for developers in development.
```

**Comment by rickhanlonii** [comment]
Just one last fix 👍

**Comment by mohamedsgap** [comment]
@rickhanlonii just made an update according to your last suggestion ;)

**Pull request merged**